crates/scroll/alloy/evm/src/block/mod.rs

```diff
@@ -174,6 +175,7 @@ where
         }
 
         // apply gas oracle predeploy upgrade at Feynman transition block.
+        #[allow(clippy::collapsible_if)]
```

Comment:
I think it looks more readable without the collapse, but let me know what you think